AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -0,0 +1,76 @@
+import requests, datetime, json, boto3, os, gzip
+
+def jc_directoryinsights(event, context):
+    jcapikey = os.environ['JCAPIKEY']
```

Comment:
I've now switched this over to handling the API key with AWS Secrets Manager but I had not previously looked into this and I would love a second set of eyes to make sure that I didn't make any glaring mistakes that would cause the Secret to not actually be secured.

Quick note: Much of the example code I saw around assumed a key-value pair but I did not think that made a ton of sense for the API key. There is a human-readable description that says what it is but I didn't see much of a need to ever reference the value when we can grab it by the Secret ID. If my thought process doesn't track, I'd love to hear.